src/tools/linter-rules.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Linter rules
+description: Details about the Dart Linter and the lint rules it provides.
+---
+
+The Dart Linter uses the analyzer to identify and report
+on "lints" found within Dart code. Linting is performed
+by the Dart analysis server through your IDE
+or with the [`dart analyze`](/tools/dart-analyze) command.
+
+This page outlines all currently existing rules,
+their use cases, what triggers them, as well as potential fixes.
+
+The rules are organized into rule groups
+which identify their type of functionality.
+
+- **[Errors](#error-rules)**
+  - Possible errors or mistakes in your code.
+- **[Style](#style-rules)**
+  - Matters of style, largely derived from the [Dart style guide][].
+- **[Pub](#pub-rules)**
+  - Possible issues with [Pub](/guides/packages) setup.
+
+Rules are also assigned a maturity level. Rules without one specified
+are considered stable, while other maturity levels are marked with the rule.
+
+- **Stable**
+  - These rules are safe to use and are verified as functional
+  with the latest versions of the Dart language.
+- **Experimental**
+  - The rules are still under evaluation and may never be stabilized.
```

Comment:
```suggestion
  - The rules are still under evaluation and might never be stabilized.
```

(See the entry for _may_ in https://developers.google.com/style/word-list#letter-m.)